sdk/selfhelp/azure-resourcemanager-selfhelp/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0-beta.1 (2023-04-28)
+
+- Azure Resource Manager selfhelp client library for Java. This package contains Microsoft Azure SDK for selfhelp Management SDK. Help RP provider. Package tag package-2023-01-01-preview. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
```

Comment:
Not sure about the service name(self-help?). Though GO is using selfhelp too, guess that's fine...
https://github.com/Azure/azure-sdk-for-go/blob/3bdc6234b14b48903e1b6c26ef15063c720c4e82/sdk/resourcemanager/selfhelp/armselfhelp/README.md

Reply:
https://github.com/Azure/azure-sdk/issues/5817  Please refer to this archborad review issue.

Reply:
Got it. I meant the manager name and service description in README. e.g. `SelfHelpManager`. You can ignore this.

Reply:
Thanks. I will add the pr to the release request issue and let service team decide.

Reply:
Yes, manager name seems better be `SelfHelp...`